interface/src/app/gene/searchFew/searchFew.js

```diff
@@ -51,17 +50,21 @@ angular.module('adage.gene.searchFew', [
               } else {
                 $scope.noResults = false;
               }
-
-              return data.results;
             },
 
             // Error function
             function(responseObject, responseHeaders) {
               $scope.errors = 'Gene search is temporarily down';
             }
-          ).$promise.then(function(response) {
-            return response.results;
-          });
+          ).$promise.then(
+            function(response) {
+              var selectedIDs = Object.keys($scope.selectedGenes);
+              var geneResultsList = response.results.filter(function(result) {
+                return selectedIDs.indexOf(result.id.toString()) === -1;
+              });
+              return geneResultsList.slice(0, numResultsToReturn);
+            }
+          );
```

Comment:
@mhuyck I agree, but I think this isn't really a bug, but rather an enhancement that can be done, since this method already returns 5 more results than what it used to (as it only shows 10 at a time).

I have thought about this, and I came up with similar implementations as the ones you were mentioning (especially the first one). In terms of the second one, yeah, we could send in the 'limit' parameter with this request here and just ask for however many genes we want. However, I think that in either case this is an enhancement that we can track as an issue and not really a bug or part of this PR. What do you think?